README.md

```diff
@@ -102,10 +102,11 @@ echo "RAILS_ENV=development" >> .env
 - Deploy the app
 
 ```shell
-docker compose -f .\docker-compose.yml -f .\docker-compose.override.init.yml up -d
+bin/deploy
```

Comment:
not needed in this PR, but the deploy shell script isn't deploying anything really - it's building and preparing. The name's a bit misleading.

Reply:
I'm not sure what you mean, this is the script we run on the droplet to deploy the app.